client/hosting/server-settings/test/index.js

```diff
@@ -178,6 +178,8 @@ describe( 'Hosting Configuration', () => {
 				dispatchEvent: jest.fn(),
 			} ) ),
 		} );
+
+		global.ResizeObserver = require( 'resize-observer-polyfill' );
```

Comment:
```suggestion
		// Mock `ResizeObserver`, which isn't currently supported in JSDOM
		Object.defineProperty( global, 'ResizeObserver', {
			writable: false,
			value: jest.fn().mockImplementation( () => ( {
				observe: jest.fn(),
				unobserve: jest.fn(),
				disconnect: jest.fn(),
			} ) ),
		} );
```
Let's remove the `resize-observer-polyfill` and do something simple like this. We don't need a true polyfill for this test, and new dependencies always introduce risk.